packages/yarn.rb

```diff
@@ -3,11 +3,9 @@
 class Yarn < Package
   description 'Yarn is a new package manager for JavaScript and an alternative to npm.'
   homepage 'https://yarnpkg.com/en/'
-  version '0.27.0-20170629.1443'
-  source_url 'https://nightly.yarnpkg.com/yarn-v0.27.0-20170629.1443.tar.gz'
-  source_sha256 'e1086d267151cd05896b15023e8c4e253a64e675a80bb38dc294b5b1c86c6656'
-
-  depends_on 'node' unless File.exists? '/usr/local/bin/node'
+  version '1.0.0-20170808.1058'
+  source_url 'https://nightly.yarnpkg.com/yarn-v1.0.0-20170808.1058.tar.gz'
+  source_sha256 '7056c5ee12aa8d014d59b377be2e494ce49a6059e2678d8f79f1fc9904eb008b'
 
```

Comment:
Agreed.  I will work on overhauling the node package to install via binary instead of compiling from source.  In the meantime, please hold off from merging this PR.